tools/klee/main.cpp

```diff
@@ -622,8 +622,12 @@ std::string KleeHandler::getRunTimeLibraryPath(const char *argv0) {
     KLEE_DEBUG_WITH_TYPE("klee_runtime", llvm::dbgs() <<
                          "Using build directory KLEE library runtime :");
     libDir = KLEE_DIR;
+#ifdef KLEE_CMAKE_BUILD
```

Comment:
I'm not sure how I feel about this. For now I'll live with it but really we need the flexibility to build the runtime with different build types.